imports/plugins/core/orders/client/components/orderTable.js

```diff
@@ -155,6 +158,13 @@ class OrderTable extends Component {
             className="rui-order-avatar"
           />
           <strong>{order.shipping[0].address.fullName}</strong> | {emailAddress}
```

Comment:
Not an issue created by this PR, and not a blocker for me on this specific PR, but very soon I think we should start rejecting PRs that don't address the `shipping[0]` or `billing[0]` issue as it's a strong indicator that that part of the code is not marketplace ready, and if we're in there, we should fix it anyway.

Reply:
those [0] are being removed/updated in #2813